api/Navigator.json

```
@@ -838,7 +838,7 @@
             },
             "firefox": {
               "version_added": "9",
-              "notes": "Before Firefox 32, <code>navigator.doNotTrack</code> would report values of <code>yes</code> and <code>no</code> rather than <code>1</code> and <code>0</code>."
+              "notes": "In Firefox, <code>navigator.doNotTrack</code> returns <code>\"unspecified\"</code> instead of <code>null</code>. Before Firefox 32, <code>navigator.doNotTrack</code> would report values of <code>\"yes\"</code> and <code>\"no\"</code> rather than <code>\"1\"</code> and <code>\"0\"</code>."
```

Since these are two distinct notes, it would be better to use the array form of "notes". I _think_ this should be something like:

```suggestion
              "notes": [
                "In Firefox, <code>navigator.doNotTrack</code> returns <code>\"unspecified\"</code> instead of <code>null</code>.",
                "Before Firefox 32, <code>navigator.doNotTrack</code> would report values of <code>\"yes\"</code> and <code>\"no\"</code> rather than <code>\"1\"</code> and <code>\"0\"</code>."
              ]
```

...but BCD is really picky about formatting JSON so I might have made a mistake here...